TACC/Core-CMS
# task/tup-551 Hide publish text from blog header

## Overview

Hide publish text from blog header.

## Related

- [TUP-551](https://tacc-main.atlassian.net/browse/TUP-551)

## Changes

- Removing "published" text from blog headers
- Adding in translations for To be published
- Added a colon in after To be published

## Testing

http://localhost:8000
1. Add in a news section
2. Publish a blog. Schedule a blog. And make one unpublished.
3. Test published blogs have no "published" text or `span` markup before their dates
4. Test when editing that "To be published" and "Unpublished" text still appears


## UI

| editing |
| - |
| <img width="648" alt="Screenshot 2023-12-14 at 12 28 59 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/f11e38d2-75bd-43a4-a161-a96235c497cd"> |

| not editing |
| - |
| <img width="562" alt="Screenshot 2023-12-14 at 12 29 25 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/f8f50569-91f2-4213-a942-3d2ea03231fe"> |
